VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -673,19 +701,25 @@ private void showVideoNotAvailable() {
         }
     }
 
-    private void hideVideoNotAvailable() {
+    private void hideVideoNotPlayInfo(VIDEO_NOT_PLAY_TYPE reason) {
```

Comment:
If you end up going with `VIDEO_NOT_PLAYING_MESSAGE` it would be consistent to name this `hideVideoNotPlayingInfo`